app/models/match_decisions/approve_match_housing_subsidy_admin.rb

```diff
@@ -111,9 +111,9 @@ def declined
         match.confirm_housing_subsidy_admin_decline_dnd_staff_decision.initialize_decision!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.